mappings/net/minecraft/util/shape/VoxelShape.mapping

```diff
@@ -59,3 +59,6 @@ CLASS net/minecraft/class_265 net/minecraft/util/shape/VoxelShape
 		ARG 11 z2
 	METHOD method_20538 getFace (Lnet/minecraft/class_2350;)Lnet/minecraft/class_265;
 		ARG 1 facing
+	METHOD method_33661 getClosestPointTo (Lnet/minecraft/class_243;)Ljava/util/Optional;
+		COMMENT {@return the closest point in the shape to {@code target}}
+		ARG 1 target
```

Comment:
When would this return `Optional.empty()`?

Reply:
When this shape is empty.